specificatie/openapi.yaml

```diff
@@ -648,6 +810,132 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Geometrie waarmee gezocht moet worden naar resources.'
+      type: object
+      required:
+        - geometrie
+      properties:      
+        geometrie:
+          $ref: '#/components/schemas/Geometrie'
+        spatialOperator:
+          type: string
+          description: >-
+            "Een operator uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1. Ondersteund worden:
+            - intersects  : object geometrie `overlapt` of `raakt` de opgegeven contour
+            - within      : object geometrie valt volledig binnen de opgegeven contour
+            - contains    : object geometrie omsluit de opgegeven contour volledig"
+          enum:
+            - intersects
+            - within
+            - contains
+          default: intersects
+        tolerantie:
+          description: 'De maximale afwijking (in meters) tussen twee punten waarbij deze punten nog als één punt worden gezien.'
+          type: number
+          format: float
+          default: 0.001
+          minimum: 0.00001
+          maximum: 1
+
+    Geometrie:
+      type: object
+      description: 'De geometrie waarmee gezocht moet worden naar objecten. Er kan een punt, cirkel of polygoon worden opgegeven.' 
+      properties:
+        locatie:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/pointGeoJSON.yaml'
+        cirkel:
+          $ref: '#/components/schemas/Cirkel'
+        polygoon:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
```

Comment:
Cirkel is volgens mij niet correct gespecificeerd. In JSON ziet het er als volgt uit:
{
    "type": "Point",
    "coordinates": [1.0, 2.0],
    "radius": 1
}
Ik verwacht
{
    "type": "circle",
    "coordinates": [1.0, 2.0],
    "radius": 1
}
of
{
    "center": {
        "type": "Point",
        "coordinates": [1.0, 2.0]
    },
    "radius": 1
}